src/test/java/net/datafaker/RandomImageGenerator.java

```diff
@@ -0,0 +1,99 @@
+package net.datafaker;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
```

Comment:
IMHO specific is better than wildcard.

Reply:
Actually I don't understand this class (`RandomImaegGenerator`) at all, it seems to be a duplicate of `Image`

Reply:
yeah, I used it for a demo, I really should have removed it. I now moved it and fixed it, it's minimal now. Thanks for feedback, I really should have reviewed my work better!